x/pairing/keeper/scores/geo_req.go

```diff
@@ -0,0 +1,216 @@
+package scores
+
+import (
+	"fmt"
+	"sort"
+
+	commontypes "github.com/lavanet/lava/common/types"
+	"github.com/lavanet/lava/utils"
+	epochstoragetypes "github.com/lavanet/lava/x/epochstorage/types"
+	"github.com/lavanet/lava/x/pairing/types"
+	planstypes "github.com/lavanet/lava/x/plans/types"
+)
+
+// geo requirement that implements the ScoreReq interface
+type GeoReq struct {
+	Geo uint64
+}
+
+const (
+	geoReqName    = "geo-req"
+	maxGeoLatency = 10000 // highest geo cost < 300
+	minGeoLatency = 1
+)
+
+// Score calculates the geo score of a provider (using the GEO_LATENCY_MAP)
```

Comment:
nit: `// Score calculates the geo score of a provider based on preset latency data`

Reply:
done. see https://github.com/lavanet/lava/commit/b5caee08e643dacb88cea147ec151e5033a28bbc